### PR TITLE
Updated Dread database with custom shields

### PR DESCRIPTION
**Pull request by steven11sjf**
- Added a "Fire storm missiles" that requires storm missiles and 3 missiles
- Added storm missile shields, bomb shields, cross bomb shields and power bomb shields to database
- Requires new ODR release, see randovania/open-dread-rando#220

### CODE REVIEW

**Comment by henriquegemignani** [comment]
You're missing the .txt versions of the db files

**Comment by steven11sjf** [comment]
This should be good to merge once #4428 is merged in and it gets a review

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**
